betty/job/__init__.py

```diff
@@ -0,0 +1,132 @@
+"""
+Provide utilities for running jobs concurrently.
```

Comment:
Document that to run jobs, three things are needed: the jobs, a context, and a scheduler.